examples/date/README.md

```diff
@@ -15,26 +15,61 @@ cd moqtransport/examples/date
 To run the server, run:
 
 ```shell
-go run *.go -server -publish
+go run . -server -publish
 ```
 
-Then, open a new shell and start a client:
+Then, open a new shell and start a client using a plain MoQ request:
 
 ```shell
-go run *.go -subscribe
+go run . -subscribe
 ```
 
-To use WebTransport, add the `-webtransport` and `-addr` flags:
+The server will also respond to webtransport requests.
 
-Server:
+To make such a request, change the client parameters to
 
 ```shell
-go run *.go -server -publish -webtransport
+go run . -subscribe -webtransport -addr https://localhost:8080/moq
 ```
 
-Client:
+## Description of the MoQ flow of this example
+
+See the [moq_communication.md](moq_communication.md) for a description of the MoQ flow.
+
+## Using fixed local certificates
+
+You can use [mkcert](https://github.com/FiloSottile/mkcert) to setup
+a local certificate authority and generate a
+self-signed certificate for the server.
+
+First, install mkcert:
+
+```shell
+brew install mkcert
+```
+
+Then, install a temporary certificate authority (will go away at reboot):
```

Comment:
Is this correct? I didn't find any info about it in the mkcert README, and I always thought it just stored the CA in the root store permanently.

Reply:
Hmm. I need to double check this since this was based on my own experience. You're probably right.

Reply:
I have removed all the examples/date documentation here and moved it into #197.
Currently without the mkcert part until I have verified it more, or found an external link.